rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -40,14 +40,40 @@
 from rotkehlchen.utils.misc import ts_sec_to_ms
 
 if TYPE_CHECKING:
+    from rotkehlchen.accounting.structures.base import HistoryBaseEntry
     from rotkehlchen.api.server import APIServer
 
 
+def assert_editing_works(
+        entry: 'HistoryBaseEntry',
+        rotkehlchen_api_server: 'APIServer',
+        sequence_index: int,
+):
+    """
+    a test to check editing any subclass of history base entry is successful
+    """
+    entry.sequence_index = sequence_index
+    entry.timestamp = TimestampMS(1639924575000)
+    entry.event_type = HistoryEventType.DEPOSIT
+    entry.asset = A_USDT
+    entry.balance = Balance(amount=FVal('1500.1'), usd_value=FVal('1499.45'))
+    entry.location_label = '0x9531C059098e3d194fF87FebB587aB07B30B1306'
+    entry.notes = 'Deposit stuff for staking somewhere'
+    entry.event_subtype = HistoryEventSubType.NONE
+    json_data = entry_to_input_dict(entry, include_identifier=True)
+    response = requests.patch(
+        api_url_for(rotkehlchen_api_server, 'historyeventresource'),
+        json=json_data,
+    )
+    assert_simple_ok_response(response)
+    assert entry.identifier is not None
```

Comment:
shouldn't you also check that the information in the db matches what you expect?